Marlin/Marlin_main.cpp

```diff
@@ -3695,122 +3688,77 @@ inline void gcode_G28() {
             SERIAL_EOL;
           }
         } //do_topography_map
-      #endif //!DELTA
-
-    #else // !AUTO_BED_LEVELING_GRID
 
-      #if ENABLED(DEBUG_LEVELING_FEATURE)
-        if (DEBUGGING(LEVELING)) SERIAL_ECHOLNPGM("> 3-point Leveling");
-      #endif
+        if (verbose_level > 0)
+          planner.bed_level_matrix.debug(" \n\nBed Level Correction Matrix:");
```

Comment:
From what I can see, `planner.bed_level_matrix` only depends on `AUTO_BED_LEVELING_FEATURE` and is not specific to grid-leveling. So I believe this block of code should still be outside of the `AUTO_BED_LEVELING_GRID` condition. However… I see that originally this block depended on `DISABLED(DELTA)`… but I'm not sure why. Now I'm thinking that it should be included for all probing styles and all kinematics.